### PR TITLE
Updated related person current name suffix patch

### PR DESCRIPTION
**Pull request by mani7002**
Description
----

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
image

image

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: lib/resources/example_json/r4_examples_relatedperson.rb [hunk 1]
```diff
@@ -789,6 +789,11 @@ module Resources
         'op': 'replace',
         'path': '/name/0/family',
         'value': 'Potter'
+      },
+      {
+        'path': '/name/0/suffix',
+        'op': 'replace',
+        'value': ['Jr.']
```
Comment (SantanuBanerjee) [inline lib/resources/example_json/r4_examples_relatedperson.rb:796]: Can the sequence be same as above, like, op, path and then value?

Comment (mani7002) [reply lib/resources/example_json/r4_examples_relatedperson.rb:796]: Done here:https://github.com/cerner/fhir.cerner.com/pull/550/commits/4946d9bb50cbe875bd66237ff6bdf3a289268f11#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR794

Comment (mani7002) [reply lib/resources/example_json/r4_examples_relatedperson.rb:796]: Done:https://github.com/cerner/fhir.cerner.com/pull/550/commits/4946d9bb50cbe875bd66237ff6bdf3a289268f11#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR794

**Review by alexbeckner1 — CHANGES_REQUESTED** [review]
Can we please have the add-identifier operation beginning on row 95 moved to the top of the list of operations? I know it's not part of this change, just wanted to keep things consistent with http://fhir.cerner.com/millennium/r4/individuals/patient/#patch

File: lib/resources/r4/related_person_patch.yaml [hunk 2]
```diff
@@ -159,3 +159,22 @@ operations:
...
+      {
+        "op": "replace",
+        "path": "/name/0/suffix",
+        "value": ["Jr."]
+      }
+    note: >
+      <ul>
+        <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `suffix` is intended to be replaced.</li>
+        <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
+        <li>If no suffix value is provided the suffix on the name will be unset.</li>
```
Comment (Coffeevee) [inline lib/resources/r4/related_person_patch.yaml:179]: nit - 'If no suffix value is provided, the suffix on the name will be unset.'

Comment (ShadabQuaiser) [reply lib/resources/r4/related_person_patch.yaml]: updated https://github.com/cerner/fhir.cerner.com/commit/2000ab899d4ad547edb36bbd886a22571e1dc726

**Comment by ShadabQuaiser** [comment]
> Can we please have the add-identifier operation beginning on row 95 moved to the top of the list of operations? I know it's not part of this change, just wanted to keep things consistent with http://fhir.cerner.com/millennium/r4/individuals/patient/#patch

done here https://github.com/cerner/fhir.cerner.com/commit/5d6fb5e6e6cc62d27af71ab18c6b0d9f5f68843e

File: lib/resources/r4/related_person_patch.yaml [hunk 3]
```diff
@@ -159,3 +159,22 @@ operations:
...
+        "op": "replace",
+        "path": "/name/0/suffix",
+        "value": ["Jr."]
+      }
+    note: >
+      <ul>
+        <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `suffix` is intended to be replaced.</li>
+        <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
+        <li>If no suffix value is provided, the suffix on the name will be unset.</li>
+      </ul>
```
Comment (sujanb12) [inline lib/resources/r4/related_person_patch.yaml:180]: let's add a new line at the end.

File: lib/resources/r4/related_person_patch.yaml [hunk 4]
```diff
@@ -159,3 +159,22 @@ operations:
...
+    type: String
+    url: https://hl7.org/fhir/R4/datatypes-definitions.html#HumanName.suffix
+    description: Replace the `HumanName.suffix` of the name at the given <code>{index}</code> in the list of related person names.
+    example: |
+      {
+        "op": "replace",
+        "path": "/name/0/suffix",
+        "value": ["Jr."]
+      }
+    note: >
```
Comment (sujanb12) [inline lib/resources/r4/related_person_patch.yaml:175]: let's make a note that only one value for the suffix is supported.

File: lib/resources/r4/related_person_patch.yaml (hunk 3 above)
Comment (ShadabQuaiser) [reply lib/resources/r4/related_person_patch.yaml]: added here https://github.com/cerner/fhir.cerner.com/commit/194a06ef023f0b84455cd6f6c0c46f35b338ee4c

File: lib/resources/r4/related_person_patch.yaml (hunk 4 above)
Comment (ShadabQuaiser) [reply lib/resources/r4/related_person_patch.yaml:175]: added here 194a06e

**Review by sujanb12 — APPROVED** [review]

**Review by Coffeevee — APPROVED** [review]

**Review by SantanuBanerjee — APPROVED** [review]

File: lib/resources/r4/related_person_patch.yaml [hunk 5]
```diff
@@ -159,3 +159,23 @@ operations:
...
+      <ul>
+        <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `suffix` is intended to be replaced.</li>
+        <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
+        <li>If no suffix value is provided, the suffix on the name will be unset.</li>
+        <li>Only one suffix value should be provided.</li>
```
[the remark this answers is not in the document]
Comment (ShadabQuaiser) [reply lib/resources/r4/related_person_patch.yaml]: done https://github.com/cerner/fhir.cerner.com/commit/ddd31de6dfc897cf22be8e246ddfed9b2105b87f

**Review by exopsRamba — APPROVED** [review]

**Review by MadhurRaj — APPROVED** [review]

**Comment by MaxPhilips** [comment]
live
Screen Shot 2021-03-05 at 12 41 24 PM

**Pull request merged**